src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node[] myTable = new Node[DEFAULT_INITIAL_CAPACITY];
+    private int size;
 
     @Override
     public void put(K key, V value) {
-
+        if (size == myTable.length * DEFAULT_LOAD_FACTOR) {
```

Comment:
add variable threshold

Reply:
I think there is no need to add a new variable in this case

Reply:
ok, leave